organized procedure into multiple sections (#108)
devfile/docs: docs/modules/user-guide/partials/proc_adding-commands-to-a-devfile.adoc

.Procedure
.Adding a command

> . Add a `commands` section in the devfile that contains a list of one or more commands.

Reviewer: ```suggestion
. Add a `commands` section to the devfile that contains a list of one or more commands.
```

+
[NOTE]

* A command can have only one action, though you can use `composite` commands to execute several commands either in sequence or in parallel.

.Adding a command group

. Assign a given command to one or more groups that represent the nature of the command.
. Use the following supported group types:
+
* `build`
* `run`
* `test`
* `debug`
+

> . For each of the groups, define one default command by specifying the `isDefault` value.

Reviewer: ```suggestion
. For each group, define a default command by specifying the `isDefault` value.
```

        isDefault: true
----

.Adding composite commands

. Chain multiple commands together by defining a composite command.
. Reference the individual commands that are called from a composite command by using the `name` of the command.

> . Specify whether commands within a composite command are to be executed in sequence or in parallel.

Reviewer: ```suggestion
. Specify whether to run the commands within the composite command in sequence or parallel.
```

> . Specify whether commands within a composite command are to be executed in sequence or in parallel.

Reviewer: ```suggestion
. Specify whether to run the commands within a composite command in sequence or parallel.
```
